### PR TITLE
PRL-4260 - Manage orders hearing screen changes

### PR DESCRIPTION
**Pull request by channa-patil**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json [hunk 1]
```diff
@@ -422,7 +412,7 @@
     "ID": "HearingData",
     "ListElementCode": "cafcassHearingChannel",
     "FieldType": "DynamicList",
-    "ElementLabel": "Cafcass",
+    "ElementLabel": "CAFCASS",
```
Comment (yogendraU) [inline definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json:415]: Please change it to Cafcass

File: definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json [hunk 2]
```diff
@@ -431,7 +421,7 @@
     "ID": "HearingData",
     "ListElementCode": "cafcassCymruHearingChannel",
     "FieldType": "DynamicList",
-    "ElementLabel": "Cafcass Cymru",
+    "ElementLabel": "CAFCASS Cymru",
```
Comment (yogendraU) [inline definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json:424]: Please change it to Cafcass Cymru

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json (hunk 1 above)
Comment (channa-patil) [reply definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json]: Done

File: definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json (hunk 2 above)
Comment (channa-patil) [reply definitions/private-law/json/ComplexTypes/HearingData/ComplexTypes.json]: Done

**Pull request closed without merging**